public/assets/js/scripts.js

```diff
@@ -265,7 +280,7 @@ let findIdeaWithMostVotes = () => {
   let groupID = location.hash.substr(1);
   let voteValArr = [];
   $.ajax({
-    url: "/api/ideas/groups/" + groupID, //CHANGED TO TAKE IN PARAMETER INSTEAD OF HARD CODED.
+    url: "/api/ideas/groups/" +  groupID, //  groupID, CHANGED TO TAKE IN PARAMETER INSTEAD OF HARD CODED.
```

Comment:
@brishaw here's the code you need here
```suggestion
    url: "/api/ideas/groups/" +  groupID, // CHANGED TO TAKE IN PARAMETER INSTEAD OF HARD CODED.
```